app/views/devise/registrations/new.html.erb

```diff
@@ -2,6 +2,20 @@
 
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+    <%= f.label :first_name %><br />
+    <%= f.text_field :first_name, autofocus: true, autocomplete: "first name" %>
+  </div>
```

Comment:
`autofocus` can only be used for one input field in a form - it makes that field active and ready for input from the user to start typing

Reply:
I removed autofocus from the 3 fields we added, first name, last name and username.